.github/workflows/main.yml

```diff
@@ -38,6 +38,16 @@ concurrency:
 
 jobs:
   # Check Code style quickly by running `rustfmt` over all code
+  Runzk:
+    name: Test generated zkwasm
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
```

Comment:
Technically, we don't really need submodules, as we don't build Rust code in this action.